src/app/src/main/res/layout/activity_accepted_book_details.xml

```diff
@@ -6,4 +6,89 @@
     android:layout_height="match_parent"
     tools:context=".AcceptedBookDetailsActivity">
 
+    <Button
+        android:id="@+id/accepted_details_action_btn"
+        android:layout_width="219dp"
+        android:layout_height="38dp"
+        android:layout_marginTop="16dp"
+        android:background="#673AB7"
+        android:onClick="handleBorrowButtonClick"
+        android:text="@string/borrow_book"
+        android:textColor="#FFFFFF"
```

Comment:
These buttons should be set to material theme like seen here: https://github.com/CMPUT301F20T01/Bookmark/pull/98/commits/096b473bbea7fae4a5c484d4ddfc37753a107f07